pkg/bootstrapper/bootstrapper.go

```diff
@@ -298,6 +310,11 @@ func (wmcb *winNodeBootstrapper) createKubeletService() error {
 		"--windows-service",
 		"--logtostderr=false",
 		"--log-file=" + filepath.Join(wmcb.installDir, "kubelet.log"),
+		// Registers the Kubelet with Windows specific taints so that linux pods won't get scheduled onto
+		// Windows nodes.
+		// TODO: Write a `against the cluster` e2e test which checks for the Windows node object created
```

Comment:
Why is this a `TODO` and not part of this PR?

Reply:
I wanted to include this as part of e2e test suite which we don't have yet.